GalaxyZooTouchTable/GalaxyZooTouchTable/Models/TableSubject.cs

```diff
@@ -0,0 +1,45 @@
+﻿using PanoptesNetClient.Models;
+
+namespace GalaxyZooTouchTable.Models
+{
+    public class TableSubject
+    {
+        public int X { get; set; }
+        public int Y { get; set; }
+        public double RA { get; set; }
+        public double DEC { get; set; }
+        public string SubjectLocation { get; set; }
+        private readonly double PlateScale = 1.5;
+        private readonly double Offset = 0.03;
+        public Subject Subject { get; set; }
+
+        public TableSubject(Subject subject, double TableRA, double TableDEC)
+        {
+            RA = System.Convert.ToDouble(subject.Metadata.ra);
+            DEC = System.Convert.ToDouble(subject.Metadata.dec);
+            Subject = subject;
+            SubjectLocation = subject.GetSubjectLocation();
+            XYConvert(TableRA, TableDEC);
+        }
+
+        private void XYConvert(double TableRA, double TableDEC)
+        {
+            int ScreenWidth = 1248; 
+            int ScreenHeight = 432; 
```

Comment:
Are these values not available to you through the code?

Reply:
They are not. There is a separation of concerns between the Views and the Viewmodels. Each `TableSubject` is constructed in the `SpaceViewModel`, but we need to know the size of the cutout width and height to correctly find out the pixel placement of each subject. 

That said, these variables should be changed to `CutoutWidth` and `CutoutHeight` to more accurately reflect what they represent.